docs/attributes.md

```diff
@@ -1,64 +1,60 @@
-* `node['splunk']['package']['file_suffix']` - URI path portion, suffix to append after building (set based on ohai attributes)
-* `node['splunk']['package']['file_name']` - Actual package file name (`"#{node['splunk']['package']['name']}#{node['splunk']['package']['file_suffix']}"`)
-* `node['splunk']['package']['url']` - Full URI to the Splunk package to download (Constructed from above package attributes)
-* `node['splunk']['package']['provider']` - Provider to use to install file (set based on ohai attributes)
-* `node['splunk']['config']['alerts']` - Data bag item used to configure alerts (`nil` - alerts not managed by chef)
-* `node['splunk']['config']['authentication']` - Data bag item used to configure authentication (`nil` - authentication not managed by chef)
-* `node['splunk']['config']['host']` - Hostname to configure the Splunk instance to report as. (EC2 Instance ID or Fully Qualified Domain Name)
-* `node['splunk']['config']['roles']` - Data bag item used to configure roles (`nil` - roles not managed by chef)
-* `node['splunk']['config']['secrets']` - Coordinate String (see [data bags documentation][data_bags]), pointing to a key within a Chef Vault encrypted data bag item used to configure the splunk.secret file. The value must be a string. (`nil` - secrets not managed by chef).  Note: this is currently not supported on windows.
-* `node['splunk']['config']['licenses']` - Data bag item that the license server recipe uses as the source of truth for the license data.
-* `node['splunk']['config']['license-pool']` - Data bag item used to configure license pools.
-* `node['splunk']['config']['ui_prefs']` - Hash of stanzas used to configure [ui-prefs.conf][] on the search head in a clustered configuration or a standalone instance.
-* `node['splunk']['config']['assumed_index']` - Name of the index to which data is forwarded to by default, when the index is not configured for the input.(`main`)
-* `node['splunk']['bootstrap_shc_member']` - Set this attribute to `true` to bootstrap a member to the Search Head Cluster (SHC). (`false`)
-* `node['splunk']['mgmt_host']` - The host other SHC members use when connecting to the current node. You probably want a wrapper cookbook to override this. (`node['ipaddress']`)
-* `node['splunk']['heavy_forwarder']['use_license_uri']` - Set this attribute to `true` to point the Heavy Forwarder to the license master. (`false`)
-* `node['splunk']['apps']` - An [apps hash](databags.md#apps-hash) of apps to configure locally. (Does not support downloading apps ... yet...)
+## Configurable (with defaults)
 
+- `node['splunk']['exclude_groups']` - Array of local groups to which the splunk user should not belong. Checking existing memberships and removing the splunk user from these groups during the chef run only works on Chef 11.10+, users of prior versions should remove the splunk user from these groups manually. (`['root','wheel','admin']`)
```

Comment:
Chef 11 isn't supported anymore is it?

Reply:
d744f3347c912df0f7e202ef85b2f8935a7b7e07